compass/Coordinator.java

```diff
@@ -202,6 +214,30 @@ private void start() throws ArgumentException, InterruptedException {
         GetTransactionsToApproveResponse txToApprove = api.getTransactionsToApprove(depth, nodeInfoResponse.getLatestMilestone());
         trunk = txToApprove.getTrunkTransaction();
         branch = txToApprove.getBranchTransaction();
+
+        if (validatorAPIs.size() > 0) {
+          boolean isConsistent = validatorAPIs.parallelStream().map(api -> {
+            CheckConsistencyResponse response = null;
+            try {
+              response = api.checkConsistency(trunk, branch);
+              if(!response.getState()) {
+                log.error("{} reported invalid consistency: {}", api.getHost(), response.getInfo());
+              }
+              return response.getState();
+            } catch (ArgumentException e) {
+              e.printStackTrace();
+              return false;
+            }
+          }).reduce(true, (a, b) -> a && b);
```

Comment:
I think `allMatch(true)` is more readable than `reduce` (& will be more efficient in the sense that enough that 1 violates it will stop).